### ISSUE TITLE
Create new logo

### ISSUE THREAD

**Issue by hagenburger** [eurucamp/livingstyleguide-eurucamp#5]
2014 was pretty nice, the same concept does not work well for 2015.

**Comment by eljojo** [comment]
You mean changing the current logo? (the one that's live on http://2015.eurucamp.org/)
I'm not a big fan of it, it's slightly confusing (looks like 2045), so I like the idea of a new one.

**Comment by hagenburger** [comment]
Yes, this issue exists because the year is hard to read. Much harder than to read than in 2014.

**Comment by bitboxer** [comment]
Please ping us if you have a new version of the logo :+1:

**Comment by bascht** [comment]
@hagenburger @bitboxer @eljojo So, we have < 4 months to eurucamp and might soon start with printing stickers / t-shirts / coffee mugs / apple watch faces - do we still want to change the actual logo? :)

**Comment by bascht** [comment]
(That point aside - I like @eljojo's idea of running it internally as eurucamp 2045 :smirk_cat:)

**Comment by hagenburger** [comment]
I’m not happy with the logo yet. I was thinking about just using the flag in our current colors. But I’m open for suggestions.

**Comment by bascht** [comment]
I like that idea. If we add the year somewhere, it would also work for black / white / print designs: ►<sup>2015</sup>
